keanu-project/src/test/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensorTest.java

```diff
@@ -210,7 +210,7 @@ public void cannotSetIfMaskLengthIsSmallerThanTensorLength() {
     @Test
     public void cannotSetIfMaskLengthIsLargerThanTensorLength() {
         DoubleTensor tensor = Nd4jDoubleTensor.scalar(3);
-        DoubleTensor mask = Nd4jDoubleTensor.create(new double[] {1., 1., 1., 1.}, new int[] {2, 2});
+        DoubleTensor mask = Nd4jDoubleTensor.create(new double[]{1., 1., 1., 1.}, new int[]{2, 2});
```

Comment:
.ones() (and we could fix that function to take a int... too?

Reply:
Good catch.